src/components/Onboarding/Screens/Apps/ConvictionVotingSettings/index.js

```diff
@@ -0,0 +1,323 @@
+import React, { Fragment, useCallback, useReducer, useState } from 'react'
```

Comment:
I like the idea @0xGabi !. There are other screens were this button may be useful as well so I think it would be better to implement it on a different PR so we maintain coherence along all the current screens